src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,9 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder originalText = new StringBuilder(text.toLowerCase()
```

Comment:
do we need to create two string builders?
let's replace `StringBuilder originalText` to `String originalText` or `text`

Reply:
done